pmoves/services/github-runner-ctl/github/client.py

```
@@ -0,0 +1,422 @@
+            GITHUB_API_REQUESTS_TOTAL.labels(endpoint=endpoint, status=status).inc()
+
+            response.raise_for_status()
+            return response.json()
+
+        except httpx.HTTPStatusError as e:
+            # Log full error details including response body for debugging
+            error_body = e.response.text[:500] if hasattr(e.response, 'text') else 'N/A'
+            logger.error(
+                f"GitHub API error: {e.response.status_code} {path} | "
+                f"Response: {error_body}"
+            )
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)